### PR TITLE
First shot at HOW TO Vault

### PR DESCRIPTION
**Pull request by jonbrohauge**
A first shot at documenting how to setup and use Vault.
Mostly setup is described here.

### CODE REVIEW

File: docs/VAULT.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Using Vault to keep tabs on environment variables
+
+## Prerequisites
+
+- An instance of [Hashcorps Vault](https://www.vaultproject.io/) running
```
Comment (x10an14) [inline docs/VAULT.md]: `Hashicorp`'s maybe =) (Lacking the 'i').

File: docs/VAULT.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Creating content in Vault can be done either using curl (not recommended) og using Vault as a client.
+A simple curl example can be found [here](vault/setup-vault-using-docker.md). Use a native vault application whenever possible. They are available for download [here](https://www.vaultproject.io/downloads.html). Using vault via docker requires you to be inside a running vault docker container. Otherwise the steps are the same.
+
+The below example assume that you have the following environment variables:
+
+- `$VAULT_TOKEN` containing the token to use as access credentials
+- `$VAULT_SERVER_URL` containing the URL to your vault server, i.e. `http://vault.domain.local:8200`
+
+If needed, enter the docker container: `docker exec -it vault /bin/sh`
```
Comment (x10an14) [inline docs/VAULT.md]: Maybe specify `library/` prefix, so as to ensure only the official Vault-image?
(If it is [this one](https://hub.docker.com/_/vault/) you were referring to).

AKA: `docker exec -it library/vault /bin/sh`

File: docs/VAULT.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Execute the following to put data in vault.
+
+```bash
```
Comment (x10an14) [inline docs/VAULT.md]: Maybe put:
```
    ```sh
```
 instead of `bash`, since you're specifying `/bin/sh` above?

File: docs/VAULT.md (hunk 1 above)
Comment (jonbrohauge) [reply docs/VAULT.md]: fixed

File: docs/VAULT.md (hunk 2 above)
Comment (jonbrohauge) [reply docs/VAULT.md]: fixed

File: docs/VAULT.md (hunk 3 above)
Comment (jonbrohauge) [reply docs/VAULT.md]: fixed

**Comment by jonbrohauge** [comment]
Argh! Didn't know your _editorial comments_ actually where changes/fixes

**Comment by x10an14** [comment]
@jonbrohauge 
> Argh! Didn't know your editorial comments actually where changes/fixes

xD Enjoy your vacation =)

**Pull request merged**